examples/image-streams/image-streams-centos7.json

```diff
@@ -22,6 +22,7 @@
               "description": "Build and run Ruby 2.0 applications",
               "iconClass": "icon-ruby",
               "tags": "builder,ruby",
+              "supports": "ruby:2.0,ruby",
```

Comment:
Hrm, I'm rethinking this.  Say I have image stream:

ruby
  2.0.x - changes for 2.x
  2.0   - intended for exact

I want to control which thing picks 2.0.  So in that case, I want supports "ruby:2.0,ruby" on tag 2.0.x and I want clients to choose it.

So:

ruby:
  2.0.x: supports: "ruby:2.0"
  2.0: supports: ""

should pick ruby:2.0.x.  So having a supports tag should weight you higher than matching image stream name and tag name exactly.

----- Original Message -----
> > @@ -22,6 +22,7 @@
> >                "description": "Build and run Ruby 2.0 applications",
> >                "iconClass": "icon-ruby",
> >                "tags": "builder,ruby",
> > +              "supports": "ruby:2.0,ruby",
> 
> I don't get the difference between "we'll first check for IS ruby and tag
> 2.0, then search IS ruby for tag 2.0" isn't that the same? But yes, I'm
> using a weighted resolver to find the right image:
> https://github.com/openshift/origin/pull/3057/files#diff-67f42376bdba60c035c6268d160059baR268
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/3057/files#r32143458